react/components/SKUSelector/components/SKUSelector.js

```diff
@@ -3,13 +3,15 @@ import React, { Component } from 'react'
 import Variation from './Variation'
 import { SKUSelectorPropTypes } from '../utils/proptypes'
 
+import styles from '../styles.css'
+
 /** Renders the main and the secondary variation, if it exists. */
 export default class SKUSelector extends Component {
   render() {
     const { selectedId, onSelectSku, mainVariation, secondaryVariation, maxSkuPrice } = this.props
 
     return (
-      <div className="vtex-sku-selector">
+      <div className={`${styles.skuSelectorContainer}`}>
```

Comment:
You can put only `className={styles.skuSelectorContainer}` instead of interpolation string.